Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Properties/ClassPropertiesExtractor.cs

```diff
@@ -24,13 +24,17 @@ public ClassPropertiesExtractor(IConverterSettingsStore<RevitConversionSettings>
 
     // add type specific props not included in parameters.
     // so far, no extra props are needed
-    /*
+    
     switch (element)
     {
+      case DB.Area area:
+        elementPropertiesDict.Add("Area Scheme", area.AreaScheme?.Name ?? "");
```

Comment:
Null values are respected. Don't need the "" fallback. `area.AreaScheme?.Name ?? ""` can just be `area.AreaScheme?.Name`. Guessing we keep the switch statement approach for future extensibility